backend/conferences/schema.py

```diff
@@ -1,11 +1,11 @@
-import graphene
+import strawberry
 
 from .models import Conference
 from .types import ConferenceType
 
 
+@strawberry.type
 class ConferenceQuery:
-    conference = graphene.Field(ConferenceType, code=graphene.String())
-
-    def resolve_conference(self, info, code):
+    @strawberry.field
+    def conference(self, info, code: str) -> ConferenceType:
```

Comment:
could we call this just "Conference"? :)

Reply:
done everywhere